_posts/0000-01-02-jasoryan.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Yesterday's the past, tomorrow's the future, but today is a gift. That's why it's called the present.